source/dub/dub.d

```diff
@@ -313,8 +313,11 @@ class Dub {
 			recipe_content = file_content[0 .. idx].strip();
 		} else throw new Exception("The source file must start with a recipe comment.");
 
+		auto nidx = recipe_content.indexOf('\n');
+
 		auto idx = recipe_content.indexOf(':');
-		enforce(idx > 0, "Missing recipe file name (e.g. \"dub.sdl:\") in recipe comment");
+		enforce(idx > 0 && (nidx < 0 || nidx > idx),
```

Comment:
In theory someone could also do something like this `/+ dub.sdl: name "foo"; dependency "bar" +/` in one line. But incidentally I was indeed also thinking of #1081. Will have a look at it in a moment.